### ISSUE TITLE
Hacktoberfest 2018

### ISSUE THREAD

**Issue by willywos** [willywos/jobbyjobjob#1]
This is a good project to work on if you want to do Hacktoberfest #Hacktoberfest Some things that might be fun to work on:

- Adding ability to login to save jobs.
- Adding your resume.
- Add stackoverflow jobs.
- Add some cool graphics that look like they are from the 80's.
- Update the layout for the job post so it looks nicer.
- Fix typos or grammer
- Add the source for the job post to the listings on the home page.

**Comment by chivoX** [comment]
@willywos is just sent a PR to Add stackoverflow jobs 👍

**Comment by willywos** [comment]
hey @chivoX yeah that was something that I recently added with a friend. I'll update this issue. Thanks

**Comment by vipulrawat** [comment]
Why don't you migrate to Javascript, it will be better when the project scales and will have good performance also.

**Comment by willywos** [comment]
@vipulrawat just personal preference I suppose. I like to use ruby on rails and I've been doing lots of javascript at work using React Native and I really missed programming with the ruby language. I'm not really looking to move it to JS. As far as scale, I don't know if I'll ever have that problem and if I did I don't think I would have much of a problem because its not in js.

 Feel free to work on this project if you want to work on one of these for hacktoberfest. 

Thanks!

**Comment by jakesorce** [comment]
@willywos - just submitted a PR to add login / save job features. Let me know what you think

**Comment by jakesorce** [comment]
@willywos - just submitted a PR to fix mobile navbar - https://github.com/willywos/jobbyjobjob/pull/6

**Comment by jakesorce** [comment]
@willywos - small grammar fixes / initial project cleanup - https://github.com/willywos/jobbyjobjob/pull/7

**Comment by willywos** [comment]
@jakesorce you da man 👊 

thank you for contributing and happy hacktoberfest!

**Comment by OvidiuToma** [comment]
Hi @willywos, I will gladly help on the "resume"  item, however I feel like we need to clarify some items.
  
What kind of features do you except?

How about(inspired by linkedin):  
- Small Bio - Text area
- Experience section - Ability to add past or present jobs(with dates and a short description)
- Skills section - Ability to add skills.


Or would you prefer a single text area that can be edited using a html editor(something like ckeditor https://ckeditor.com/ckeditor-5/)

**Comment by OvidiuToma** [comment]
@willywos I see that the project has authentication implemented, how about authorization? Are you familiar with the "cancan" gem? I could add it and configure the permisions for the existing controllers.

**Comment by OvidiuToma** [comment]
@willywos I see that the project has authentication implemented, how about authorization? Are you familiar with the "cancan" gem? I could add it and configure the permisions for the existing controllers.

**Comment by willywos** [comment]
@OvidiuToma Yeah I'm familiar with cancan, if you want to take that on that would be great. Thanks!

**Comment by willywos** [comment]
Thank You Everyone!
